### ISSUE TITLE
[FEATURE] Add health check script or EP to connectors

### ISSUE THREAD

**Issue by MaxwellDPS** [OpenCTI-Platform/connectors#906]
## Use case

To be able to allow kubernetes to proactively identify any containers in an unhealthy state

## Current Workaround

General bash hacks, doing a curl for the api

## Proposed Solution

Add an API endpoint or a script that can be used to validate the health of the container natively 

## Additional Information

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

## If the feature request is approved, would you be willing to submit a PR?

Yes

**Comment by Jipegien** [comment]
@sbocahu is it still relevant in the current situation?

**Comment by sbocahu** [comment]
@Jipegien yes, AFAIK at the time we have no such health check at the connector container level.
An HTTP lifeness endpoint could be implemented in the pycti (https://github.com/OpenCTI-Platform/client-python) python module, returning the connector state I suppose
